### ISSUE TITLE
bug: include_raw_content not working when using Tavily API

### ISSUE THREAD

**Issue by rere950303** [assafelovic/gpt-researcher#401]
When I turn off the `include_raw_content` option, I get 5 results.
```
{
    "query": "네이버웍스에서 메일 예약 전송 방법 알려줘",
    "follow_up_questions": null,
    "answer": null,
    "images": null,
    "results": [
        {
            "title": "메일 사용 가이드 - 메일 - WORKS Help Center",
            "url": "https://guide.worksmobile.com/kr/mail/mail-guide/",
            "content": "네이버웍스 메일은 기업에서 안전하게 사용할 수 있는 보안과 스팸 메일 차단 기술이 적용되어 있고, 비즈니스 효율을 높일 수 있는 풍부한 기능을 제공합니다. 메일함 받은 편지함, 보낸 편지함, 임시 보관함, 메모함, 내 메일함, 스팸 메일함, 휴지통 등의 기본 메일함 이외에 전체메일, VIP메일, 안 ...",
            "score": 0.9789,
            "raw_content": null
        },
        {
            "title": "WORKS Help Center",
            "url": "https://guide.worksmobile.com/kr/",
            "content": "네이버웍스 상품 가입 방법을 알려주세요. 네이버웍스 사칭으로 의심되는 메일을 받았어요. 비밀번호를 잊어버려서 로그인을 할 수 없습니다. 메일이 들어오지 않습니다.(수신 실패) 각 서비스의 데이터는 어떻게 백업할 수 있나요? 재택 근무 도입",
            "score": 0.96294,
            "raw_content": null
        },
        {
            "title": "수신 확인 및 발송 취소 - 메일 - WORKS Help Center",
            "url": "https://guide.worksmobile.com/kr/mail/mail-guide/sending-email/check-read-receipt-and-cancel-mail/",
            "content": "받는 사람의 메일 주소가 본인이 속한 네이버웍스의 메일 계정인 경우에 메일을 보낸 후 보낸 메일을 취소할 수 있습니다. 예약(대기) 메일은 발송이 되지 않았다면 예약을 취소할 수 있습니다. ... 가장 도움이 되지 않았던 점을 알려주세요.",
            "score": 0.95497,
            "raw_content": null
        },
        {
            "title": "발송 예약 - 메일 - WORKS Help Center",
            "url": "https://guide.worksmobile.com/kr/mail/mail-guide/sending-email/schedule-send-mail/",
            "content": "메일 쓰기 화면에서 오른쪽 상단의 아이콘을 탭합니다. '발송 예약'을 탭합니다. 추천된 시간 옵션 또는 '직접 설정'을 탭하여 원하는 날짜와 시간을 선택합니다. '전송'을 탭해 메일 발송을 예약합니다. 메일 제목 아래에 발송 예약 날짜와 시간을 확인할 수 ...",
            "score": 0.94952,
            "raw_content": null
        },
        {
            "title": "메일 쓰기 - 메일 - WORKS Help Center",
            "url": "https://guide.worksmobile.com/kr/mail/mail-guide/sending-email/write-mail/",
            "content": "메일쓰기 화면 오른쪽 위의 아이콘을 탭할 경우. 메모 쓰기: 메모쓰기로 전환합니다. 발송 예약: 추천 시간 옵션을 선택하거나 원하는 날짜와 시간을 예약하여 메일을 발송합니다. 개인별 발송: 여러 주소로 보내더라도 받는사람은 본인의 계정만 표시되어 ...",
            "score": 0.94462,
            "raw_content": null
        }
    ],
    "response_time": 0.88
}
```

But when I turn on the `include_raw_content` option, I only get one result.

```
{
    "query": "네이버웍스에서 메일 예약 전송 방법 알려줘",
    "follow_up_questions": null,
    "answer": null,
    "images": null,
    "results": [
        {
            "title": "메일 쓰기 - 메일 - WORKS Help Center",
            "url": "https://guide.worksmobile.com/kr/mail/mail-guide/sending-email/write-mail/",
            "content": "메일쓰기 화면 오른쪽 위의 아이콘을 탭할 경우. 메모 쓰기: 메모쓰기로 전환합니다. 발송 예약: 추천 시간 옵션을 선택하거나 원하는 날짜와 시간을 예약하여 메일을 발송합니다. 개인별 발송: 여러 주소로 보내더라도 받는사람은 본인의 계정만 표시되어 ...",
            "score": 0.91088,
            "raw_content": "메일 쓰기\nCONTENTS\n이 기능은 Basic, Premium 상품에서만 사용할 수 있습니다.\n메일에는 텍스트뿐만 아니라 사진, 동영상, 설문, 파일, 스티커 등을 첨부할 수 있습니다.\n메일 작성 중 발송 예약을 하거나  아이콘을 탭하여 메모 쓰기로 변환하는 것도 가능합니다.\n메일을 보내는 기본적인 방법은 다음과 같습니다.\n모바일\nPC웹\n새 메일을 쓰다가 다른 메일이나 일정을 열 수 있고, 임시 보관함에 저장 후 이어 쓸 수 있습니다.\n메일 쓰기 항목\n모바일\n메일쓰기 화면의 입력 또는 설정 항목\n제목 오른쪽의\n아이콘을 탭할 경우 (Android)\n제목 오른쪽의\n아이콘을 탭할 경우 (iOS)\n메일쓰기 화면 오른쪽 위의\n아이콘을 탭할 경우\nPC웹\n메일쓰기 화면의 입력 또는 설정 항목\n메일쓰기 화면 상단에 표시되는 아이콘\n메일쓰기 화면 상단에 표시되는 메뉴\n받는 사람, 참조, 숨은참조 지정\n받는사람, 참조, 숨은참조에 사람을 지정할 수 있습니다.\nPC웹의 경우, 사람 이름을 입력할 때 자동완성 목록 하단의 '프로필 이미지 표시'를 선택하면 프로필 사진이 표시됩니다.\n받는 사람이나 참조, 또는 숨은 참조에 외부 메일 주소가 1개 이상 포함되어 있을 경우, 메일 쓰기 화면에 보안에 유의하라는 배너가 노출됩니다.\n관리자의 설정에 따라서 메일 발송 직전에 보안 경고 대화창이 노출될 수도 있습니다. 단, 보내는 사람이 외부 메일 주소가 포함된 메일링리스트(DL)일 때는 보안 경고 대화창이 노출되지 않습니다.\n한번 메일을 보낼 때 외부 메일 주소는 500개까지 추가할 수 있습니다.\n받는 사람 지정\n회사 주소록과 개인 주소록에 등록된 구성원이나 메일링리스트, 그룹명 등을 검색하거나 직접 이메일 주소를 입력할 수 있습니다.\n참조, 숨은참조 지정\n모바일\nPC웹\n조직도에서 검색하기\n조직도에서 직접 구성원을 선택하여 받는사람, 참조, 숨은참조로 지정할 수 있습니다.\n모바일\nPC웹\n받는 사람 주소의 색깔\n받는 사람의 주소에는 다음 색깔이 지정됩니다.\n본문 편집\n텍스트 스타일 변경\n본문의 글자 크기, 모양, 색깔을 변경할 수 있습니다.\n모바일\nPC웹\n사진 삽입\n기기에 있는 사진을 불러오거나 카메라로 촬영해서 본문에 사진을 삽입할 수 있습니다.\n모바일\nPC웹\n관리자가 설정한 파일 용량 이하의 사진만 본문에 삽입할 수 있습니다.\n사진 편집\n본문에 삽입한 사진을 편집할 수 있습니다.\nPC웹\n사진 편집 후 원본으로 돌리고 싶다면\n아이콘을 클릭합니다.\n스티커 삽입\n모바일\nPC웹\n설문 삽입\n작성한 내 설문을 본문에 삽입할 수 있습니다.\n모바일\n본문 번역 삽입\n작성중인 본문을 다른 언어로 번역 후 본문에 삽입할 수 있습니다.\n모바일\nPC웹\n번역 지원 언어는 영어, 일본어, 중국어(간체/번체), 프랑스어, 베트남어, 태국어, 인도네시아어, 스페인어입니다.\n링크 삽입\n본문 텍스트에 링크를 삽입할 수 있습니다.\nPC웹\n링크 편집 및 삭제\n본문의 텍스트에 삽입된 링크를 편집하거나 삭제할 수 있습니다.\nPC웹\n파일 첨부\n기기에 있는 사진 및 동영상 파일 첨부\n기기에 있는 사진 및 동영상을 파일로 첨부할 수 있습니다.\n모바일\nPC웹\n드라이브에 있는 파일 첨부\n드라이브에 있는 사진 및 동영상 또는 문서 파일을 첨부할 수 있습니다.\n모바일\nPC웹\n첨부 파일 최대 개수와 최대 용량\n메일 작성시 첨부할 수 있는 일반 첨부 파일 개수는 100개입니다. 일반 첨부의 최대 용량은 관리자가 설정할 수 있습니다.\n일반 첨부를 초과하는 경우 다운로드 기한이 있는 파일로 자동 변환됩니다.\nPC웹에서는 파일 첨부 시 일반 첨부를 다운로드 기한이 있는 파일로 수동전환할 수도 있습니다.\n다운로드 기한이 있는 파일의 용량과 개수는 관리자가 설정한 값에 따릅니다.\n메모 쓰기\n메일에서 메모 쓰기로 전환해 다양한 형식으로 아이디어를 기록할 수 있습니다.\n모바일\nPC웹\n그 외 불편했던 점을 알려주세요.\nCONTENTS"
        }
    ],
    "response_time": 2.35
}
```

Is it a bug? I want raw content to come down as a response for all 5 results.

**Comment by rere950303** [comment]
@assafelovic thanks for your works!
could you check this issue please?

**Comment by assafelovic** [comment]
@rere950303 have you checked other sites? This might be due to failure of scraping the sites

**Comment by rere950303** [comment]
@assafelovic 
When I tested it on another site, I checked that the response came out normally. 
When I tested it, I set `Include Domains` to `guide.worksmobile.com` , but when I turned off the `include_raw_content` option, it seems that scraping is normal because the `https://guide.worksmobile.com/kr/mail/mail-guide/sending-email/schedule-send-mail/` result came out, but I was wondering if the result would be truncated in relation to the score.
I just want raw content for `https://guide.worksmobile.com/kr/mail/mail-guide/sending-email/schedule-send-mail/` out of the results when turning off the `include_raw_content` option. If scraping fails for `https://guide.worksmobile.com/kr/mail/mail-guide/sending-email/schedule-send-mail/` , shouldn't it be included in the results when turning off the `include_raw_content` option?
